.php_cs

```diff
@@ -0,0 +1,33 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . "/src");
+
+return PhpCsFixer\Config::create()
+    ->setRules(
+        [
+            '@PSR2'                               => true,
+            'array_syntax'                        => [
+                'syntax' => 'short'
+            ],
+            'class_keyword_remove'                => false,
+            'concat_space'                        => [
+                'spacing' => 'one'
+            ],
+            'combine_consecutive_unsets'          => true,
+            'general_phpdoc_annotation_remove'    => [
+                'annotations' => ['@author'],
+            ],
+            'linebreak_after_opening_tag'         => true,
+            'no_short_echo_tag'                   => true,
+            'no_useless_else'                     => true,
```

Comment:
I did not expect this feedback/response, I would suggest you to see the video of @rdohms of his talk "Writing code that lasts" … or writing code you won’t hate tomorrow. Where Rafael clearly and visually explains what benefits to achieve.
- https://youtu.be/rMW6MZIREgg?t=32m5s (4 minutes)